webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -374,7 +401,11 @@ def update_corr_index(
         triggered_id = ctx.triggered[0]["prop_id"].split(".")[0]
 
         if triggered_id == parent.uuid("parameter-selection-store-resp"):
-            return {response: corr_xindex.get(response, 0) for response in responses}
+            default_index = 0
```

Comment:
I tried to let explicit that we have a default value because it took me more than a second to understand how it worked :D